cleanlab/classification.py

```diff
@@ -379,17 +452,21 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
                     "Assigning sample weights for final training based on estimated label quality."
                 )
 
-            self.sample_weight = np.ones(np.shape(labels_cleaned))
+            sample_weight = np.ones(np.shape(labels_cleaned))
             for k in range(self.num_classes):
                 sample_weight_k = 1.0 / max(self.noise_matrix[k][k], 1e-3)  # clip sample weights
-                self.sample_weight[labels_cleaned == k] = sample_weight_k
+                sample_weight[labels_cleaned == k] = sample_weight_k
 
+            sample_weight_expanded = np.zeros(len(labels))  # pad pruned examples with zero weight
+            sample_weight_expanded[x_mask] = sample_weight
+            self.label_issues_df["sample_weight"] = sample_weight_expanded
+            self.sample_weight = self.label_issues_df["sample_weight"]
```

Comment:
So, you're changing the functionality here right? Seem okay to store the expanded version instead, but maybe add a comment here above "Store the sample weight for every example in the original, unfiltered dataset"

Reply:
yes storing expanded weights is more useful IMO. added the comment.